projects/control-service/projects/helm_charts/pipelines-control-service/templates/deployment.yaml

```diff
@@ -34,6 +34,10 @@ spec:
         # https://helm.sh/docs/howto/charts_tips_and_tricks Automatically Roll Deployments
         checksum/secrets: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
     spec:
+      initContainers:
```

Comment:
This will impact the production code. 
DO you want that? 
IMO opinion no.

Reply:
It takes some 30 sec for the Postgres (more time than CockroachDB) to be up and VDK waits for it, the probe fails and causes a restart of the VDK deployment. That's the only reason why I added this init to avoid the restart.

Reply:
tested it now with the latest changes without this dependency, the vdk dep pod restarted 3 times. If we are fine with such restarts, I can drop this.

Reply:
This can't be in production code though. 
will the production code find the database? 
Is the restarts not ok as eventually it gets running? 
Can you remove it and check does the start eventually work ?

Reply:
makes sense, it will just do at most 2-3 restarts and then go up, but it eventually goes up so I can drop this